requirements.txt

```diff
@@ -9,7 +9,9 @@ pint>=0.9
 matplotlib>=2.1.1
 vtk>=7.1.1
 PyOpenGL>=3.1.0
-mayavi>=4.5.0
+mayavi>=4.5.0 # for mayavi-visualizer
+python-vtk # for mayavi-visualizer, e.g. python3-vtk7 (Ubuntu)
+python-wxgtk # for mayavi-visualizer, e.g. python3-wxgtk4.0 (Ubuntu)
```

Comment:
there is no package `python-wxgtk` on pypi (https://pypi.org/project/python-wxgtk/), please remove it

`python-vtk` is a personal project with no documentation nor repository (https://pypi.org/project/python-vtk/), which is then used with `import python_vtk`; are you really sure this is what we should install?

Reply:
I removed that change to `requirements.txt`.

Reply:
Fine with me, thanks!